.github/workflows/release.yaml

```diff
@@ -2,10 +2,8 @@ name: Release Charts
 
 on:
   push:
-    branches:
-      - main
```

Comment:
Do you still need the branch here? If only on main?

Reply:
branch doesn't matter here so I'm removing it. this will run on each tag push regardless of branch which allow for testing prior to PR.